pkgs/tools/misc/pika/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchurl
+, stdenv
+, undmg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pika";
+  version = "0.0.12";
+
+  src = fetchurl {
+    url = "https://github.com/superhighfives/${pname}/releases/download/${version}/Pika-${version}.dmg";
```

Comment:
Can it be build from sources?

Reply:
In theory, it could be. But, I have no experience on building an xcodeproj. I tried to build it with xcbuildhook. It segfaults.
```
/nix/store/3m12kq2gfxsszqqhcfcfj6r4n4dxx2qj-hook/nix-support/setup-hook: line 1:  6020 Segmentation fault: 11  xcodebuild SYMROOT=$PWD/Products OBJROOT=$PWD/Intermediates $xcbui>
```